Source/SwiftLintBuiltInRules/Rules/Lint/FunctionArgumentsSpacingRule.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FunctionArgumentsSpacingRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "functions_arguments_spacing",
+        name: "Function Arguments Spacing",
+        description: "Remove the space before the first function argument and after the last argument",
+        kind: .lint
+    )
+}
+
+private extension FunctionArgumentsSpacingRule {
+    final class Visitor: ViolationsSyntaxVisitor<ConfigurationType> {
+        override func visitPost(_ node: FunctionCallExprSyntax) {
+            let argsCount = node.arguments.count
+            guard argsCount != 0 else {
```

Comment:
The case of no parameters could also be interesting if you consider for instance `f( )`.